api/maintainer/owned-packages.js

```diff
@@ -15,7 +15,12 @@ module.exports = async (req, res, ctx) => {
     const cleanPackages = packages.map(({
       id, name, language, registry, donationRevenue, adRevenue
     }) => ({
-      id, name, language, registry, donationRevenue, adRevenue
+      id,
+      name,
+      language,
+      registry,
+      donationRevenue: donationRevenue ? donationRevenue.reduce((acc, r) => (acc += r.amount), 0) : 0,
```

Comment:
we made this a number in our schema since we don't want to return the UI a whole bunch of data that we might not want exposed, so this was being returned as undefined since it was an array not a number